src/foam/u2/view/AnyView.js

```diff
@@ -54,6 +54,14 @@ foam.CLASS({
               return foam.Object.isInstance(o) ? o : {};
             }
           }),
+          foam.u2.view.AnyView.Choice.create({
+            label: 'FObject',
+            type: foam.core.FObject,
+            view: foam.u2.view.FObjectView,
+            toType: function(o) {
+              return foam.core.FObject.isInstance(o) ? o : {};
```

Comment:
```suggestion
              return foam.core.FObject.isInstance(o) ? o : foam.core.FObject.create();
```

Reply:
That makes sense; I committed this change. I think right now this case will never happen, but it would be an awfully subtle bug if this changed in the future.